lib/csv/writer.rb

```diff
@@ -149,5 +152,9 @@ def quote(field)
         end
       end
     end
+
+    def apply_converters_to(row, lineno)
+      @converters.convert(row, nil, lineno)
+    end
```

Comment:
Could you remove this method and use `@converters.convert` directly?
It looks needless abstraction to me.

Reply:
I added this abstraction because I thought it would increase readability, but I agree it's a little bit overkill in this scenario.